test/hotspot/jtreg/runtime/cds/CDSMapReader.java

```diff
@@ -282,16 +309,29 @@ public static int validate(MapFile mapFile) {
                     //     To see how an invalidate pointer may be found, change oop in the
                     //     following line to oop+1
                     mustContain(mapFile.oopToObject, field, oop, false);
-                    count ++;
+                    count1 ++;
```

Comment:
Could `oop` be 0? If so, `oop` could be overcounted.

Pre-existing: extra import at line 27
`import java.io.RandomAccessFile;`

Reply:
I added checks for `oop != 0`, so now the test is counting only the non-null references.